src/es/components/molecules/openingHours/OpeningHours.js

```diff
@@ -0,0 +1,121 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+/**
+* @export
+* @class OpeningHours
+* @type {CustomElementConstructor}
+*/
+export default class OpeningHours extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+    if (this.shouldRenderHTML()) this.renderHTML()
+  }
+
+  disconnectedCallback () {}
+
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldRenderCSS () {
+    return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
+  }
+
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldRenderHTML () {
+    return !this.badge
+  }
+
+  /**
+   * renders the css
+   */
+  renderCSS () {
+    this.css = /* css */`
+        * {
+            box-sizing: border-box;
+            margin: 0;
+            padding: 0;
+        }
+        table {
+            text-align: left;
+            width: 100%;
+            max-width: 320px;
```

Comment:
Maybe it is good to introduce a variable here with hardcoded 320px as fallback.

Reply:
Alles klar, würde ich auf der anderen branch noch machen von der Standortdetail